www/src/pages/en/usage/env-variables.mdx

```diff
@@ -85,7 +91,7 @@ if (parsed.success === false) {
 
 Then, we use a proxy object to throw errors if you try to access a server-side environment variable on the client.
 
-```ts:env.mjs
+```ts
```

Comment:
Is the reason for removing the filenames here that the border of the code component is broken inside of `<details>`?

I think it's better to keep it with the filename but a broken border, and fix the broken border in a future PR.

(there's more than one of these, so this comment is for all of them)

Reply:
Yeah, that's the reason. Also, the broken border means having few bars with `env.mjs` randomly placed on top of the doc which can be kind of annoying. I can add the names back and show you how it looks tomorrow

Reply:
if its really bad, maybe a temp fix is to put the filename as a comment on the first line?

Reply:
Actually, you can see the bug in a deploy here: https://create-t3-caezf6nlb-t3-oss.vercel.app/en/usage/env-variables
I actually have no clue why it's happening because changing the page back to `.md` file does not fix it :shrug: 
I updated the pr with comments with file names